app/src/main/java/com/github/se/project/ui/components/ExpandableLessonSection.kt

```diff
@@ -91,21 +91,26 @@ fun ExpandableLessonSection(
                   if (!isInstant) {
                     {
                       Row(verticalAlignment = Alignment.CenterVertically) {
-                        if (lessonCountText.isNotEmpty() && lessons.isNotEmpty()) {
-                          Text(
-                              text = lessonCountText + " lessons",
-                              style = MaterialTheme.typography.bodyMedium)
-                          Spacer(modifier = Modifier.width(8.dp))
-                        }
-                        if (lessons.isNotEmpty()) {
-                          IconButton(onClick = { expanded = !expanded }) {
-                            Icon(
-                                imageVector =
-                                    if (expanded) Icons.Default.KeyboardArrowDown
-                                    else Icons.AutoMirrored.Filled.KeyboardArrowLeft,
-                                contentDescription = if (expanded) "Collapse" else "Expand")
+                        if (lessonCountText.isNotEmpty()) {
+                          if (lessons.size > 1) {
+                            Text(
+                                text = "$lessonCountText lessons",
+                                style = MaterialTheme.typography.bodyMedium)
+                            Spacer(modifier = Modifier.width(8.dp))
+                          } else {
+                            Text(
+                                text = "$lessonCountText lesson",
```

Comment:
great implementation of the coach suggestion. much better to have "lesson" instead of "lessons" when there is only 1 lesson.